batavia/core/version.js

```diff
@@ -6,7 +6,9 @@ var version = {}
 var magic_map = {}
 magic_map[String.fromCharCode(22, 13, 13, 10)] = [[3, 5], [0xF]]
 magic_map[String.fromCharCode(23, 13, 13, 10)] = [[3, 5, 3], [0xF]]
-magic_map[String.fromCharCode(51, 13, 13, 10)] = [[3, 6, 2], [0xF]]
+magic_map[String.fromCharCode(51, 13, 13, 10)] = [[3, 6], [0xF]]
+magic_map[String.fromCharCode(66, 13, 13, 10)] = [[3, 7], [0xF]]
+magic_map[String.fromCharCode(73, 13, 13, 10)] = [[3, 8], [0xF]]
```

Comment:
Could this reference the core constants, rather than re-creating the String.fromCharCode calls?

Reply:
Sweet.